src/internal/Subscriber.ts

```diff
@@ -158,16 +156,7 @@ export class Subscriber<T> extends Subscription implements Observer<T> {
 
   /** @deprecated This is an internal implementation detail, do not use. */
   _addParentTeardownLogic(parentTeardownLogic: TeardownLogic) {
```

Comment:
We can probably just remove _addParentTeardownLogic entirely.

1. It's prefixed with `_` which is a universal signal for "do not use".
2. It's been marked as "deprecated do not use" since the moment it was added.

Reply:
My concern is that the name has been used in [the `isTrustedSubscriber` test](https://github.com/ReactiveX/rxjs/blob/6.3.2/src/internal/Subscriber.ts#L328-L330) and that's been released. I wasn't entirely sure of what the affect of it's removal in a new release would be. I suppose it means a trusted subscriber could end up being re-wrapped in a using-6.3.3-with-6.3.2 scenario.